app/events/adjustment_event.rb

```diff
@@ -0,0 +1,14 @@
+class AdjustmentEvent < Event
+  # @param adjustment [Adjustment]
+  def self.publish(adjustment)
+    self.create(
+      eventable: adjustment,
+      organization_id: adjustment.organization_id,
+      event_time: Time.zone.now,
+      data: EventTypes::InventoryPayload.new(
+        items: EventTypes::EventLineItem.from_line_items(adjustment.line_items, to: adjustment.storage_location_id)
+      ).as_json
```

Comment:
Why do some of these have `.as_json` and others don't?

Reply:
Which ones don't?